roadmap/2022/MLOpsRoadmap2022.md

```diff
@@ -40,21 +40,47 @@ Almog Baku, Rimoto <almog.baku@gmail.com>
 
 ### What is MLOps?
 
-MLOps could be narrowly defined as "the ability to apply DevOps principles to Machine Learning applications" however as we shall see shortly, this narrow definition misses the true value of MLOps to the customer. Instead, we define MLOps as “the extension of the DevOps methodology to include Machine Learning and Data Science assets as first class citizens within the DevOps ecology”.
+MLOps is often narrowly defined as:
+>*The ability to apply DevOps principles to Machine Learning applications.*
 
-MLOps should be viewed as a practice for consistently managing the ML aspects of products in a way that is unified with all of the other technical and non-technical elements necessary to successfully commercialise those products with maximum potential for viability in the marketplace. This includes DataOps, too, as machine learning without complete, consistent, semantically valid, correct, timely, and unbiased data is problematic or leads to flawed solutions that can exacerbate built-in biases.
+This narrow definition misses the true value of MLOps to the data scientist and engineers supporting data scientists.
 
+MLOps should be viewed as a practice for consistently managing the ML aspects of products in a way that is unified with all of the other technical and non-technical elements necessary to successfully commercialise those products with maximum potential for viability in the marketplace. 
 
-> MLOps is not to be confused with "AIOps". AIOps often means an application of AI technologies to Ops data with sometimes unclear aims for gaining insights. These terms are still evolving, but for the purposes of this document we do not mean AIOps in the latter usage. Some organisations are more comfortable with the designation 'AI' rather than 'Machine Learning' and so it is to be expected that MLOps may be referred to by AIOps in those domains, however the reverse is not true as use of the term 'AIOps' may not refer to the MLOps methodology.
+Accordingly, we believe a better definition of MLOps is:
+> *The extension of the DevOps methodology to include Machine Learning and Data Science Assets and the interdependencies between those Assets as first class citizens within the DevOps ecology.*
 
+Some ML & DS Assets require new extensions to DevOps practices:
+* Features [1]
+* Trained Models [2]
+
+Some ML & DS Assets can be handled by existing software DevOps practices, but must be linkable to the other ML Assets
+* Model Servers [3]
+* Model Training Pipelines [4]
+

```

Comment:
I suggest we'll mention that these new resources should obviously be linked to lower-level DevOps resources such as computing, storage, etc.